src/webui/index.js

```diff
@@ -69,16 +68,6 @@ const createWindow = () => {
   return window
 }
 
-// Converts a Multiaddr to a valid value for Origin HTTP header
-const apiOrigin = (apiMultiaddr) => {
-  // Return opaque origin when there is no API yet
-  // https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque
-  if (!apiMultiaddr) return 'null'
-  // Return the Origin of HTTP API
-  const apiUri = toUri(apiMultiaddr, { assumeHttp: true })
-  return new URL(apiUri).origin
-}
-
```

Comment:
:information_source: yep, this is no longer needed due to https://github.com/ipfs/go-ipfs-cmds/pull/201